linux_os/guide/system/accounts/accounts-session/file_permission_user_init_files/bash/shared.sh

```diff
@@ -6,9 +6,9 @@
 
 {{{ bash_instantiate_variables("var_user_initialization_files_regex") }}}
 
-readarray -t interactive_users < <(awk -F: '$3>={{{ uid_min }}}   {print $1}' /etc/passwd)
-readarray -t interactive_users_home < <(awk -F: '$3>={{{ uid_min }}}   {print $6}' /etc/passwd)
-readarray -t interactive_users_shell < <(awk -F: '$3>={{{ uid_min }}}   {print $7}' /etc/passwd)
+readarray -t interactive_users < <(awk -F: '$3==0 || $3>={{{ uid_min }}}   {print $1}' /etc/passwd)
```

Comment:
This always checks root do we want that?

Reply:
I think yes - I think that is the point of the issue that the root's files were not evaluated and were not remediatied.